inference-engine/src/mkldnn_plugin/mkldnn_node.h

```diff
@@ -477,6 +510,14 @@ class MKLDNNNode : public InferenceEngine::details::no_copy {
         return desc.outputNumbers();
     }
 
+    const PerfCounters & perfCounters() const {
+        return profiling;
+    }
+
+    PerfCounters & perfCounters() {
```

Comment:
What is a purpose of this method? I do not see code which use this property getter/setter. The single consumer of this `profiling` field is a `MKLDNNGraph` but it already is in friends.

Reply:
This method is used in nodes registrar to modify the counters.